script.sh

```diff
@@ -0,0 +1 @@
+..
```

Comment:
kya khali do dots daal je chordh diye yaro

Reply:
git kill demo22

Reply:
e ka horiya hai

Reply:
"please"

Reply:
https://gifdb.com/gif/the-office-michael-scott-steve-carell-eye-stare-e1gmtq1hy9rt3d2h.html